_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/scripts/js/rhd/terms-and-conditions.js

```diff
@@ -62,8 +62,8 @@ app.termsAndConditions = {
 
         //create template string for thankyou message
```

Comment:
This html string isn't handled the same way as referrer.js.  Both are a bit much to read.  Is there a better way to inject the html in both cases?  If you'd rather it go into a new story, I'd agree.  It's possible we can get the html from a config object passed to drupalSettings (json), so that the contents are stored in the db somewhere?

Reply:
Agree is hard to read, had not refactored as not part of story requirement. I think a new story would be a good way to move forward, possibly look at using es6 as that has much better html variable handling (https://wesbos.com/template-strings-html/)